### ISSUE TITLE
Definition for JSON Keys for each Key Value Pair

### ISSUE THREAD

**Issue by Aniformer** [usnationalarchives/federalregister-api-core#9]
Hi team, 

The API output returns more than 40 key value pairs at times depending on the package issue and it's corresponding granule. Is there any documentation provided that defines the key value pairs extracted? There are a lot of keys for which we don't know the definition or meaning of and it would be really helpful if you could provide us with the definitions or perhaps a link to the documentation.

**Comment by peregrinator** [comment]
Can you provide a url for the API endpoint you are referring to? All FederalRegister.gov API endpoints are described here: https://www.federalregister.gov/developers/api/v1

**Comment by Aniformer** [comment]
https://www.federalregister.gov/api/v1/documents/2018-27875.json 

This is the API endpoint that I was referring to. Providing us a definition for these fields would really be helpful!

**Comment by peregrinator** [comment]
We don't currently have such a document available - out general design approach is to make those data field as self explanatory as possible if you are familiar with Federal Register documents. Are there particular fields in which you are unsure about?

**Comment by Aniformer** [comment]
Yes, these are the list of fields we are unsure about that appear as an output to the above API endpoint : 

1. comment_url
2. comments_close_on
3. correction_of
4. corrections
5. docket_ids
6. proclamation_number
7. regulation_id_number_info
8. regulation_id_numbers
9. regulations_dot_gov_info
10. significant
11. disposition_notes
12. page_length
13. page_views
14. signing_date
15. start_page
16. subtype
17. toc_subject
18. toc_doc
19. type 
20. volume

We're looking to understand what do these keys contain, and what their corresponding values entail. Thank you.
